blockchain/v0/reactor.go

```diff
@@ -209,7 +209,7 @@ func (bcR *BlockchainReactor) respondToPeer(msg *bcproto.BlockRequest,
 func (bcR *BlockchainReactor) Receive(chID byte, src p2p.Peer, msgBytes []byte) {
 	msg, err := bc.DecodeMsg(msgBytes)
 	if err != nil {
-		bcR.Logger.Error("Error decoding message", "src", src, "chId", chID, "msg", msg, "err", err, "bytes", msgBytes)
+		bcR.Logger.Error("Error decoding message", "src", src, "chId", chID, "msg", msg, "err", err)
```

Comment:
```suggestion
		bcR.Logger.Error("Error decoding message", "src", src, "chId", chID, "err", err)
```

msg is nil, so no need to print it too

Reply:
oh nice, good catch